Expensify/App
# fix: LoginPage for wide screens

### Details

Use KeyboardAvoidingView to push the input into the viewport and Scrollview to Allow the user to scroll the screen.

### Fixed Issues

$ https://github.com/Expensify/App/issues/5159

### Tests 
1. Open the Login page on a tablet or Ipad.
2. Click on login input and see it remains in the View and is not hidden by Keyboard.
---
1. Open the Password Page on Tablet or Ipad.
2. Clicked all the inputs to see if they are not hidden behind the keyboard.
---
1. Open the Login page on landscape Mode.
2. Clicked all the inputs to see if they are not hidden behind the keyboard.

### QA Steps
1. Open Login, Password pages on Landscape Mode or tablet or Ipad.
2. Observe the input should not be hidden behind the keyboard.


### Tested On


### Screenshots


#### Web | Desktop

![image](https://user-images.githubusercontent.com/24370807/144914964-c6c11549-0f1a-4eee-b858-92bb3f46b5b7.png)

#### Mobile Web

![Screenshot 2021-12-07 01:42:23](https://user-images.githubusercontent.com/24370807/144915608-ab76dc8c-2475-4a3b-b900-b8b316ec9925.png)

#### iOS


#### Android

![Screenshot 2021-12-07 01:37:36](https://user-images.githubusercontent.com/24370807/144914994-70447c12-bce2-400e-8f89-a6855be1f7d7.png)

https://user-images.githubusercontent.com/24370807/144915095-08fd1ae3-215c-4130-ad09-66cb71d808cc.mp4
